### PR TITLE
Make the credit smaller, and remove the slogan

### PR DESCRIPTION
**Pull request by rick-nu**
### What

This PR removes the pla intoduction line when running the pla command, and moved the credits to the "running target" line instead.
### Why

Personally I think this is cleaner this way, plus the user already knows why he is using pla (or at least, I'm pretty sure of it).

**Note**: Feel free to shoot this down, as it is personal preference.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Editors
+.idea/
```
Comment (rtuin) [inline .gitignore]: Please put this in a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

**Comment by rtuin** [comment]
There are more paths to the output of the line you moved. For example, when you call Pla in a wrong way, you now see:

```
$ pla
Pla 0.6.2 by Richard Tuin - Coder's simplest workflow automation tool.
Usage: pla [OPTIONS] [TARGET]

Error: Invalid value: Target "all" not present in Plafile.yml.
Valid targets are:
    release
    distribute
```

After your change it is:

```
$ pla
Usage: pla [OPTIONS] [TARGET]

Error: Invalid value: Target "all" not present in Plafile.yml.
Valid targets are:
    release
    distribute
```

As you see it's missing the version and tool name/author.

All error/exit paths can be found by looking for `raise` in the `pla/pla.py` file.

**Comment by rick-nu** [comment]
@rtuin I coudn't find a way to fix that properly, so all I did now was leave the extra enter out :wink:

**Comment by rtuin** [comment]
Thanks for your input on this Rick, but I don't think it adds value.

**Comment by rick-nu** [comment]
Fair enough

**Pull request closed without merging**